### DISCUSSION TITLE
config menu style

### DISCUSSION THREAD

**Discussion by icegood** [doublecmd/doublecmd#1618] [Suggestions]
### Version

Double Commander
Version: 1.1.14 gamma
Revision: 468
Commit: edd44b0e2
Build date: 2024/05/04
Lazarus: 2.2.6.0
FPC: 3.2.2
Platform: x86_64-Linux-gtk2
OS version: Ubuntu 24.04 LTS

Widgetset library: GTK 2.24.33

### Description

Config menu has an old style. Should be hardly refactored at least with 'search settings' with fuzzy search. It is impossible to remember which page is responsible for which option.

**Comment by Skif-off** [comment]
> It is impossible to remember which page is responsible for which option.

It seems, section/page names appear meaningful and appropriate and in many cases we can guess which category of settings is needed...

A filter by parameter name is available at the bottom of the "Options" window, have you tried it?
